### DISCUSSION TITLE
HELP!  How do I get STRETCHLEY OFF my Mac???  TIA!

### DISCUSSION THREAD

**Discussion by mdfloydyess** [community/community#129653] [Programming Help]
### Select Topic Area

Question

### Body

HELP! I am not a programmer or even technical. HOW DO I GET STRETCHLEY OFF my Mac???  TIA
I couldn't even find a person to contact.

**Comment by LiteBrite82** [comment]
You can try this [YouTube tutorial](https://www.youtube.com/watch?v=4WfPAdhL35E) by easyOSX

**Comment by mdfloydyess** [comment]
Perhaps I wasn't clear when I said I was not technical, so "helpful" things
like

"*Checking for `sudo` access (which may request your password)...*

Password:"


mean nothing to me, so I've wasted half an hour trying these various
"helpful" tips. I appreciate the thought, but . . .
----------------------------------------------------------
margueritefloyd.com
----------------------------------------------------------

**Reply by ebndev** [reply]
Hi @mdfloydyess,

If it's an application, you should be able to open your Applications folder and move it to the trash bin to uninstall. 

If that was unsuccessful, the developer has all of the ways you can contact them on the bottom of their [website](https://hovancik.net/stretchly/) and that might be more helpful!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by mdfloydyess** [comment]
thank you; got it!
----------------------------------------------------------
margueritefloyd.com
----------------------------------------------------------
